rust/src/filetracker.rs

```diff
@@ -105,6 +106,7 @@ impl FileTransferTracker {
             self.file.file_close(config, &self.track_id, self.file_flags);
         }
         self.file_open = false;
+        self.file_closed = true;
```

Comment:
should we set this in `trunc()` below as well?

Reply:
It is not needed because `file_is_truncated` is checked in `is_initialized` and `trunc` does not set `file_open` to false

Reply:
Ok, thanks.